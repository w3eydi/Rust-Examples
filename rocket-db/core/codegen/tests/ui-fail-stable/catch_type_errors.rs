../ui-fail/catch_type_errors.rs