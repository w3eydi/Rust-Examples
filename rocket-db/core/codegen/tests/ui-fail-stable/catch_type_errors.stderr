error[E0277]: the trait bound `usize: Responder<'_, '_>` is not satisfied
 --> $DIR/catch_type_errors.rs:6:30
  |
6 | fn f1(_request: &Request) -> usize {
  |                              ^^^^^ the trait `Responder<'_, '_>` is not implemented for `usize`
  |
  = note: required by `respond_to`

error[E0277]: the trait bound `bool: Responder<'_, '_>` is not satisfied
  --> $DIR/catch_type_errors.rs:11:30
   |
11 | fn f2(_request: &Request) -> bool {
   |                              ^^^^ the trait `Responder<'_, '_>` is not implemented for `bool`
   |
   = note: required by `respond_to`

error[E0308]: mismatched types
  --> $DIR/catch_type_errors.rs:16:17
   |
16 | fn f3(_request: bool) -> usize {
   |                 ^^^^ expected `bool`, found `&rocket::Request<'_>`

error[E0277]: the trait bound `usize: Responder<'_, '_>` is not satisfied
  --> $DIR/catch_type_errors.rs:16:26
   |
16 | fn f3(_request: bool) -> usize {
   |                          ^^^^^ the trait `Responder<'_, '_>` is not implemented for `usize`
   |
   = note: required by `respond_to`

error[E0277]: the trait bound `usize: Responder<'_, '_>` is not satisfied
  --> $DIR/catch_type_errors.rs:21:12
   |
21 | fn f4() -> usize {
   |            ^^^^^ the trait `Responder<'_, '_>` is not implemented for `usize`
   |
   = note: required by `respond_to`
